index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./main.css">
+</head>
+
+<body>
+    <div id="main">
+        <div id="header">
+            <h1>Dr. Norman Borlaug</h1>
+            <p>The man who saved a billion lives</p>
+        </div>
+
+
+        <div id="img-div">
+            <img id="image"
+                src="https://images.unsplash.com/photo-1613837433169-d74597534a69?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTR8fGRyLiUyMG5vcm1hbiUyMGJvcmxhdWd8ZW58MHx8MHx8&auto=format&fit=crop&w=500&q=60"
+                alt="image" />
```

Comment:
The image should not have atl attribute named image name it something else to allow screen readers clarify what to say.